docs/install.md

```diff
@@ -53,7 +53,39 @@ The `gnmic` container image can be pulled from Dockerhub or GitHub container reg
 ```bash
 # pull latest release from dockerhub
 docker pull gnmic/gnmic:latest
+# pull a specific release from dockerhub
+docker pull gnmic/gnmic:0.7.0
 
-# pull the specific release from github registry
+# pull latest release from github registry
+docker pull ghcr.io/karimra/gnmic:latest
+# pull a specific release from github registry
 docker pull ghcr.io/karimra/gnmic:0.5.2
 ```
+
+Example running `gnmic get` command using the docker image:
+```bash
+docker run \
+       --network host \
+       --rm ghcr.io/karimra/gnmic get --log --username admin --password admin --insecure --address router1.local --path /interfaces
+```
+### Docker Compose
```

Comment:
should the compose example have a section with a node that is launched alongside?